librz/analysis/meta.c

```diff
@@ -139,6 +139,7 @@ static bool meta_set(RzAnalysis *a, RzAnalysisMetaType type, int subtype, ut64 f
 }
 
 RZ_API bool rz_meta_set_string(RzAnalysis *a, RzAnalysisMetaType type, ut64 addr, const char *s) {
```

Comment:
add RZ_OWN, RZ_NONNULL etc..

Reply:
It turned out to be nullable; just rechecked the logic as for string, it doesn't move or be "borrowed", thus leaving it without any specific mark.